swo_aws_extension/flows/steps/configure_apn_program.py

```
@@ -0,0 +1,87 @@
+
+    def __init__(self, config):
+        self._config = config
+
+    @override
+    def pre_step(self, context: PurchaseContext) -> None:
+        phase = get_phase(context.order)
+        if phase != PhasesEnum.CONFIGURE_APN_PROGRAM:
+            raise SkipStepError(
+                f"{context.order_id} - Next - Current phase is '{phase}', skipping as it"
+                f" is not '{PhasesEnum.CONFIGURE_APN_PROGRAM}'"
+            )
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)